fragment.go

```diff
@@ -1750,15 +1750,11 @@ func (f *fragment) importRoaring(data []byte, clear bool) error {
 
 	if clear {
 		bm = f.storage.Difference(bm)
+	} else if f.storage.Containers.Size() >= bm.Containers.Size() {
+		f.storage.UnionInPlace(bm)
+		bm = f.storage
 	} else {
-		if cnt := f.storage.Count(); cnt > 0 {
-			if incomingCnt > int(cnt) {
```

Comment:
We can probably drop the computation of incomingCnt, since we're dropping the usage of it. (Compiler doesn't catch this because += is a use-and-store, so it doesn't notice that the last stored value is never used.)